test.html

```diff
@@ -0,0 +1,19 @@
+<html>
+    <head></head>
+    <body>
+        <script>
+            var showLiveChatButton  = true;
+            var showVideoChatButton = true;
+            var scaleDroneChannelId = 'fygLrCqVZQCdZfs6';
+            var nicBusNumber        = '4599130';
+            var nicChatPOC          = '1605d414-489c-4df4-83b1-334dbeb0ab2c';
```

Comment:
This is a public repository. Looks like too many keys have been added in the commit :thinking:

Reply:
Thank you. I was under the impression that these are not sensitive, but removing them makes sense.
 FYI I've used `bfg` to remove the file from the git history
FYI the credentials in `mian.js` doc section are not real!